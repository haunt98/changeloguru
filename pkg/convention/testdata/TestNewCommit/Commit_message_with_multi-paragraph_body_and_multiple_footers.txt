fix: correct minor typos in code

see the issue for details

on typos fixed.

Reviewed-by: Z
Refs #133
