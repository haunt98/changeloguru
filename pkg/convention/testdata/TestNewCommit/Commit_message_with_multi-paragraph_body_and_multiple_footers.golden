{
  "Type": "fix",
  "Scope": "",
  "Description": "correct minor typos in code",
  "BodyAndFooters": "see the issue for details\n\non typos fixed.\n\nReviewed-by: Z\nRefs #133"
}